data/models/diagnostic.py

```diff
@@ -1685,12 +1685,16 @@ def appro_badge(self):
             # * 100 to get around floating point errors when we are on the cusp
             if bio_share * 100 >= bio_threshold and combined_share * 100 >= combined_threshold:
                 return True
+        if self.tunnel_appro:
+            return False
 
     @property
     def waste_badge(self):
```

Comment:
```suggestion
    def waste_badge(self) -> bool | None:
```
une façon de rendre ça visible pourrait être avec du "typing"